### PR TITLE
Remove duplicate code

### PR DESCRIPTION
**Pull request by Timmy109**

### CODE REVIEW

File: src/main/java/dev/austech/betterreports/utils/Common.java [hunk 1]
```diff
@@ -30,9 +30,9 @@
 import org.bukkit.configuration.file.FileConfiguration;
 
 public final class Common {
-    private static String logPrefix = "&8[&c&l&oBetter&4&l&oReports&r&8] &f";
 
     public static void log(String message){
+        String logPrefix = "&8[&c&l&oBetter&4&l&oReports&r&8] &f";
```
Comment (benwoo1110) [inline src/main/java/dev/austech/betterreports/utils/Common.java:34]: Why move this here tho?

Comment (Timmy109) [reply src/main/java/dev/austech/betterreports/utils/Common.java:34]: intelliJ was like wassup this may be final

Comment (mnewt00) [reply src/main/java/dev/austech/betterreports/utils/Common.java:34]: its only used once, it doesn't even really need to be a var

Comment (Timmy109) [reply src/main/java/dev/austech/betterreports/utils/Common.java:34]: True that

File: src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java [hunk 2]
```diff
@@ -37,6 +37,8 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static dev.austech.betterreports.commands.AdminCommand.helpCommand;
```
Comment (mnewt00) [inline src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java:40]: Why is this imported as static? It is unsafe.

Comment (mnewt00) [inline src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java:40]: Why is this imported as static? It is unsafe.

File: src/main/java/dev/austech/betterreports/utils/Common.java (hunk 1 above)
Comment (mnewt00) [reply src/main/java/dev/austech/betterreports/utils/Common.java:34]: Move to inline variable as used once

File: src/main/java/dev/austech/betterreports/commands/ReportBugCommand.java [hunk 3]
```diff
@@ -50,60 +50,59 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
...
-            eb.setColor(Color.decode(Common.getConfig().getString("discord-embed-bug-report-colour")));
-            eb.setFooter("BetterReports - Timmy109", "");
-            webhook.addEmbed(eb);
-
-            try {
-                webhook.execute();
-            } catch (Exception ex) {
-                ex.printStackTrace();
-                sender.sendMessage(Common.color("&cError sending the bug report to discord. Please contact the admin."));
+    static boolean helpCommand(CommandSender sender, String[] args, List<String> adminHelp, List<String> playerHelp) {
```
Comment (mnewt00) [inline src/main/java/dev/austech/betterreports/commands/ReportBugCommand.java:96]: Move to Common.java, also don't put functions embedded deeply in classes

File: src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java (hunk 2 above)
Comment (mnewt00) [reply src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java:40]:
```suggestion
import dev.austech.betterreports.commands.AdminCommand.helpCommand;
```

File: src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java [hunk 4]
```diff
@@ -50,15 +52,7 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
...
-		if (args.length == 0) {
-			if (sender.hasPermission("betterreports.admin")) {
-				adminHelp.forEach(s -> sender.sendMessage(Common.color(s)));
-				return true;
-			}
-
-			playerHelp.forEach(s -> sender.sendMessage(Common.color(s)));
-			return true;
-		}
+		if (helpCommand(sender, args, adminHelp, playerHelp)) return true;
```
Comment (mnewt00) [inline src/main/java/dev/austech/betterreports/commands/ReportPlayerCommand.java:55]:
```suggestion
		if (AdminCommand.helpCommand(sender, args, adminHelp, playerHelp)) return true;
```

**Review by mnewt00 — CHANGES_REQUESTED** [review]

**Comment by Timmy109** [comment]
I can do better lel

**Pull request closed without merging**